README.md

```diff
@@ -13,3 +13,222 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+
+//
+
+
+Name: Kane Kriz
+
+Start Date: 11 Feb 2025
+
+Feedback Request 1: 12 Feb 2025
+
+
+//
+
+
+Feedback Request 2: 13 April 2025
+
+
+//
+
+
+
+Response 1: 
+
+
+**Reason 1: Disregard for lower terms**
+
+The first issue with asymptotic analysis being potentially misleading is due to a fundamental part of asymptotic analysis as a whole.
+
+Asymptotic analysis focuses upon the highest order and most important term in regards to performance.
+
+However, smaller order terms still make an impact in realistic scenarios without notably large (asymptotic) input sizes.
+
+In varying situations, these smaller order terms still can make a difference to the overall performance and runtime, though with asymptotic analysis, they are intentionally ignored.
+
+// Below section is the remnanats of the now scrapped second reason that just ended up being too similar to the first every time
+// ...kept them as they still have relevance to reason 1 but you can disregard them.
+
+In actuality, real world performance has the potential to vary dramatically due to more nuanced factors that are disregarded when simply considering asymptotics and shared highest order.
+
+Two algorithms could have the exact same asymptoptic runtime but behave with notable variation relative to one another in practice.
+
+Asymptotic analysis could give off the impression that these two algorithms sharing the same listed asymptotic runtime should be running near identical in common applications, but this is not (often) the case.
+
+
+
+
+
+**Reason 2: **
+
+
+Asymptotic analysis could be potentially misleading as it assumes basic operations such as traditional compares to act in constant O(1) time.
+
+For instance, consdier comparisons between high bit integers and comparisons between low bit integers.
+
+A similar scenario can be considered when thinking about comparisons between short length strings and comparisons between long length strings.
+
+In asymptotic analysis, these basic operations are assumed to be performed in constant time.
+
+However, these comparisons still increase in required work and thus runtime as their size / complexity increases.
+
+By treating these operations as constants within asymptotic analysis, further differences between expectations and real world runtime can occur.
+
+
+
+
+
+
+**Reason 3: Defined Case Types being Restrictive, Non Informative Regarding differing likelihood of best and worst cases potentially occuring**
+
+
+A third misleading issue about asymptotic analysis is that one of three scenarios are always considered: worst case, average case, or best case scenario.
```

Comment:
I don't think that this reason is going in the right direction. It all seems to come down to the analysis not being done correctly. Please also remember that best/worst/average case analysis are *independent* of asymptotic analysis -- the cases give us an expression that we can then determine an asymptotic bound for.